app/feeds/new/index/template.hbs

```diff
@@ -28,7 +28,7 @@
 	</div>
 	<br>
 	<div class="container email-caption">
-		After this feed is added to Transitland, our servers will periodically check this URL for updates
+		After this feed is part of the Feed Registry, the Transitland servers will periodically check this website for updates.
```

Comment:
Could we stick with "added" -- "After this feed is added to the Feed Registry..."?